types/mindustry/tslint.json

```diff
@@ -0,0 +1,10 @@
+{ 
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules":{
+	"no-unnecessary-class":false,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/mindustry/tslint.json;
  Line: 4;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.